src/components/existing-request/existing-request-search.vue

```diff
@@ -94,6 +102,19 @@
             </v-col>
           </v-row>
 
+          <!-- SIXTH LINE -->
+          <v-row v-if="false" class="mt-3" no-gutters> <!-- Condition to be: getIsAuthenticated -->
+            <v-col class="text-center">
+              <span
```

Comment:
Why use a span instead of a button?

Reply:
Because `V-btn` in name request was rigidly styled.. probably my own doing/blunder. 
I think you improved this in your last PR. I will pull that in and apply the updated styling if possible 👍